### DISCUSSION TITLE
BackendAPIError(400 Bad Request)

### DISCUSSION THREAD

**Discussion by JHyunB** [lablup/contribution-academy-2021#65] [Q&A]
**Describe the problem**
client-py에서  backend.ai run python -c "print('Hello World\!')"를 입력했을 때, BackendAPIError가 발생합니다.

**To Reproduce**
cd /home/jonghyun/backend.ai/backend.ai-dev/client-py
source my-backend-ai.sh
backend.ai run python -c "print('Hello World\!')"

**Expected behavior**
Hello world! 출력

**Screenshots**
1.  manager
![image](https://user-images.githubusercontent.com/36959292/130021988-f0503c54-2318-4f89-a9e9-e3f3c0ea0748.png)

2. agent
![image](https://user-images.githubusercontent.com/36959292/130022071-b8fd6eaa-d8ba-4599-8aeb-8c51abc04789.png)

3. storage-proxy
![image](https://user-images.githubusercontent.com/36959292/130022114-b53bb352-b301-4433-aef2-711da06d3b60.png)

4. web server
![image](https://user-images.githubusercontent.com/36959292/130022155-fa228da6-86fd-4812-b0d0-db5d3dc3695f.png)

5. client-py
![image](https://user-images.githubusercontent.com/36959292/130022235-98993b21-e23d-4e5b-8f8d-b6ea72b48a51.png)

**Environment Description**
OS : WSL2 Ubuntu 20.04 LTS
Current shell : bash

**Comment by youngjun0627** [comment]
혹시 커맨드로 `backend.ai admin images` 치신 후에
![image](https://user-images.githubusercontent.com/57058726/130025650-2b00b088-26bf-4cd7-bc59-c964ad1c2f2a.png)

와 같이 alias 되었는지 확인해보시겠어요??

**Reply by JHyunB** [reply]
There are no registered images라고 나옵니다!
![image](https://user-images.githubusercontent.com/36959292/130025951-79ea6b2a-7029-419d-a91c-e29eac115ceb.png)

**Reply by youngjun0627** [reply]
그럼 `backend.ai admin rescan-images` 쳐보시고 다시 `backend.ai admin images`쳐서 확인해보세요

**Reply by JHyunB** [reply]
![image](https://user-images.githubusercontent.com/36959292/130028978-af03b1d7-23eb-49e0-b4df-ac58de931141.png)
위와 같습니다

**Reply by lizable** [reply]
@JHyunB please show me the result after executing the command below.
`backend.ai mgr etcd get --prefix config`

**Reply by lizable** [reply]
I guess something went wrong during executing some scripts in `install-dev.sh`...
In [this part](https://github.com/lablup/backend.ai/blob/98f70434582e5a938d59b8d90fd99e752fd93397/scripts/install-dev.sh#L643-L650), I presume.

**Reply by JHyunB** [reply]
@lizable 말씀하신 부분 설치 로그를 보니 아래와 같이 에러가 발생합니다.
![image](https://user-images.githubusercontent.com/36959292/130047408-82d673ba-114a-4ba1-bdd9-4fede2e8d89d.png)
원인은 아직 파악하지 못했습니다..

**Reply by lizable** [reply]
@JHyunB did you enabled WSL integration in docker desktop Windows? And It seems you installed python 3.9, not python 3.9.5, is it correct?
